$ANDROID_SDK/licenses/android-sdk-license

```diff
@@ -0,0 +1 @@
+-e "\n8933bad161af4178b1185d1a37fbf41ea5269c55" 
```

Comment:
what is this and why is it necessary? same for android sdk preview license

Reply:
hmm good question, I'll look into it :)

Reply:
removed it!